### DISCUSSION TITLE
Just forked a proxy server that can be hosted on Render to work with this.

### DISCUSSION THREAD

**Discussion by preinstallable** [SanjaySunil/BetterDiscordPanel#328] [Show and tell]
Yes, it's forked from roproxy, but the code was simple enough that i could hack it into this.

You can self-host this and you can use it to bypass the useragent restriction. Personally i used a spoofer extension, but dang this is easier.

https://github.com/evanultra01/proxy-test-thingy

Now to get the server to work in this, you fork this repo and set up pages for it, then go into the discord.min.js in assets/js and replace the discord.com/api a bit down the file with yourproxyname.onrender.com/api. It's that easy. For anyone reading this, since i'm lazy as screw, please don't piggyback off of other's servers. It's not okay to do that and it wastes resources for freeloaders like me who set up their own servers for their own uses. Make your own.

**Comment by TheMin3s** [comment]
I tried using the file to create a proxy server on Render. It asked me for my payment information. Is there any way to circumvent this?

**Reply by preinstallable** [reply]
I think deploying it to Render just asks for payment info but you can just close it out since it’s free. It’s been a while since i last deployed this, so i might be wrong. Be sure to change the proxy server url in the code to your domain on Render.

**Reply by preinstallable** [reply]
Also, the payment is only if you go beyond a certain limit of requests for the month which for Render is WAY more than you should probably ever use alone unless your usage is getting gobbled up by people who will inevitably find this project. Like multiple thousands.

**Reply by TheMin3s** [reply]
Oh. I guess it may have been because of the latter.
